src/EditorFeatures/Test2/Rename/RenameEngineResult.vb

```diff
@@ -87,7 +83,10 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
                 Dim locations = Renamer.FindRenameLocationsAsync(
                     workspace.CurrentSolution, symbol, optionSet, CancellationToken.None).Result
 
-                Dim result = locations.ResolveConflictsAsync(renameTo, nonConflictSymbols:=Nothing, cancellationToken:=CancellationToken.None).Result
+                Dim result = locations.ResolveConflictsAsync(renameTo, nonConflictSymbols:=Nothing, cancellationToken:=CancellationToken.None).GetAwaiter().GetResult()
+                If result.ErrorMessage IsNot Nothing Then
+                    Throw New ArgumentException(result.ErrorMessage)
```

Comment:
>Throw New ArgumentException(result.ErrorMessage) [](start = 20, length = 48)

Contract.ThrowIfFalse

Reply:
Or rather, shouldn't the error be part of the results?

---
In reply to: [413956910](https://github.com/dotnet/roslyn/pull/43573#discussion_r413956910) [](ancestors = 413956910)

Reply:
Sure. I can do that in a follow-up or :)

Reply:
Done inhttps://github.com/dotnet/roslyn/pull/43592